Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM ubuntu:16.04
+FROM quay.io/biocontainers/metfrag:2.4.2--1
+#FROM ubuntu:16.04
 
-LABEL software.version=2.4.3
+LABEL software.version=2.4.3-abernichtwirklich
```

Comment:
Caveat: Bioconda still has only 2.4.2 Not sure if we need 2.4.3 in Phnmnl.

Reply:
For a newer MetFrag in Bioconda we need:

1. Install a newer Jarfile, bump version number (and change to https, while we're at it)
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/metfrag/meta.yaml#L6
1. copy the jar where it belongs in https://github.com/bioconda/bioconda-recipes/blob/master/recipes/metfrag/build.sh#L7

and then a PR for the Bioconda people.